evap/evaluation/templates/base.html

```diff
@@ -181,8 +181,11 @@
 
             // activate clickable hover tables
             document.querySelectorAll(".table-seamless-links [data-url]").forEach(row => {
-                row.addEventListener("click", () => {
-                    window.location.assign(row.dataset.url);
+                row.addEventListener("click", event => {
+                    // We navigate via the <a> tag to enable browser behavior for Ctrl-Click.
+                    const anchor = document.createElement("a");
+                    anchor.href = row.dataset.url;
+                    anchor.dispatchEvent(new PointerEvent("click", event));
```

Comment:
What's going on here, why don't just forward `event`? I understand PointerEvent may be preferrable to MouseEvent, but I would have hoped that we wouldn't need to care in this context. We're just handling `click` and forwarding to `click`.

Reply:
For `anchor.dispatchEvent(event)` I get `Uncaught DOMException: An attempt was made to use an object that is not, or is no longer, usable`. I makes some sense, because `event` is tied to `row` via `event.target` and has state about event-bubbling and whatnot.

I then picked PointerEvent based on https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event#event_type. If we are not confident with that (for example because of the Firefox disclaimer on the mdn page), I am also happy with `new event.constructor(event.type, event)` (from https://stackoverflow.com/a/20541207/13679671)

Reply:
Ah, makes sense. Weird, the MDN site I checked earlier listed "MouseEvent" as the event class for "click".

I'm fine with any approach, with a slight tendency towards
```javascript
// `event` has target-dependent state (`target`, bubbling, ...) -> create a new one
anchor.dispatchEvent(new event.constructor(event.type, event));
```